pkg/ebpf/c/tracer-ebpf.c

```diff
@@ -214,71 +214,33 @@ static int guess_offsets(tracer_status_t* status, struct sock* skp) {
         return 0;
 
     tracer_status_t new_status = {};
+    // Copy values from status to new_status
+    bpf_probe_read(&new_status, sizeof(tracer_status_t), status);
```

Comment:
Ah haha - this works?

Reply:
This actually reduces the number of BPF generated instructions, which is great :)

Reply:
it does! `memcpy` eBPF-certified ™️